Tests/Fixtures/Generated/Strings/flat-swift4/localizable-customName.swift

```diff
@@ -60,8 +60,6 @@ extension XCTLoc {
 
 // swiftlint:disable convenience_type
 private final class BundleToken {
-  static let bundle: Bundle = {
-    Bundle(for: BundleToken.self)
-  }()
+  static let bundle = Bundle(for: BundleToken.self)
```

Comment:
Yeah always wondered why our templates used an immediately-evaluated-closure here instead of the expression directly like you did. Probably some legacy or some preference from a past contributor, not sure. Good change 👍